src/components/_globalDataFetching/index.tsx

```diff
@@ -2,17 +2,15 @@
 import React, { useEffect } from "react";
 import { useDispatch } from "react-redux";
 
-import { getAllCategories } from "@/store/thunks/listings/categories";
-import { getAllListings } from "@/store/thunks/listings/listings";
-import { getTypeOfPlace } from "@/store/thunks/listings/typeOfPlace";
+// import { getTypeOfPlace } from "@/store/thunks/listings/typeOfPlace";
 
 export const GlobalDataOnLoad = () => {
   const dispath = useDispatch();
   useEffect(() => {
     Promise.all([
       // dispath(getAllCategories() as any),
-      dispath(getTypeOfPlace() as any),
-      dispath(getAllListings() as any),
+      // dispath(getTypeOfPlace() as any),
```

Comment:
looks like you can totally remove useEffect and rest code which you used inside useEffect